tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/NewApprenticeshipProvider/HidePassedNotificationTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿using Dfc.CourseDirectory.WebV2.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Dfc.CourseDirectory.WebV2.Tests.FeatureTests.NewApprenticeshipProvider
+{
+    public class HidePassedNotificationTests : TestBase
+    {
+        public HidePassedNotificationTests(CourseDirectoryApplicationFactory factory)
+            : base(factory)
+        {
+        }
+
+        [Fact]
+        public async Task Post_InvalidSubmissionReturnsBadRequest()
+        {
+            // Arrange
+            var providerId = await TestData.CreateProvider(apprenticeshipQAStatus: ApprenticeshipQAStatus.Passed);
+            await User.AsHelpdesk();
+
+            // Act
+            var response = await HttpClient.PostAsync(
+                $"new-apprenticeship-provider/hide-passed-notification?providerId={providerId}&returnUrl=/",null);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+        }
+
+        [Theory]
+        [InlineData(ApprenticeshipQAStatus.Submitted)]
+        [InlineData(ApprenticeshipQAStatus.InProgress)]
+        [InlineData(ApprenticeshipQAStatus.Failed)]
+        [InlineData(ApprenticeshipQAStatus.UnableToComplete)]
+        public async Task Post_NonePassedQAStatusReturnsBadRequest(ApprenticeshipQAStatus status)
```

Comment:
Nit: `NotPassed` instead of `NonePassed`

Reply:
done.